apps/src/aiTutor/chatApi.ts

```diff
@@ -6,11 +6,14 @@ import {
   ChatCompletionMessage,
 } from '@cdo/apps/aiTutor/types';
 import HttpClient from '@cdo/apps/util/HttpClient';
-import {CHAT_COMPLETION_URL} from './constants';
 import Lab2Registry from '../lab2/Lab2Registry';
 
+const CHAT_COMPLETION_URL = '/openai/chat_completion';
```

Comment:
I moved this out of the `/aichat/constants` file because it's only used here, and the generative AI team requested we move this `chatApi` file into the aiTutor directory because they're going to write a new API for themselves.